CONTRIBUTING.md

```diff
@@ -54,3 +55,9 @@ $ git push fork feature
 ```
 
 Open a pull request against the main opencensus-go repo.
+
+## General Notes
+This project uses Go 1.11.4 and Travis for CI.	This project uses Appveyor and Travis for CI.
```

Comment:
nit: remove the first "and Travis for CI".

Reply:
Done.